models/src/test/scala/coop/rchain/models/rholang/SortTest.scala

```diff
@@ -500,6 +504,77 @@ class ParSortMatcherSpec extends FlatSpec with Matchers {
     result.term should be(sortedParMatch)
   }
 
+  it should "sort Lets based on declarations and then body" in {
+    /*
+      let x /\ y = 5 in { "Test" } | let x = 4, y = 5 in { 4 } | let x = true, y = 5 in { false } ==>
+      let x = true, y = 5 in { false } | let x = 4, y = 5 in { 4 } | let x /\ y = 5 in { "Test" }
+     */
+    val parLet = Par(
+      lets = List(
+        Let(
+          declarations = List(
+            Declaration(
+              Connective(ConnAndBody(ConnectiveBody(List(EVar(FreeVar(0)), EVar(FreeVar(1)))))),
+              GInt(5),
+              freeCount = 2
+            )
+          ),
+          body = GString("Test"),
+          bindCount = 2
+        ),
+        Let(
+          declarations = List(
+            Declaration(EVar(FreeVar(0)), GInt(4), freeCount = 1),
+            Declaration(EVar(FreeVar(1)), GInt(5), freeCount = 1)
+          ),
+          body = GInt(4),
+          bindCount = 2
+        ),
+        Let(
+          declarations = List(
+            Declaration(EVar(FreeVar(0)), GBool(true), freeCount = 1),
+            Declaration(EVar(FreeVar(1)), GInt(5), freeCount = 1)
+          ),
+          body = GBool(false),
+          bindCount = 2
+        )
+      )
+    )
+    val sortedParLet = Par(
+      lets = List(
+        Let(
+          declarations = List(
+            Declaration(EVar(FreeVar(0)), GBool(true), freeCount = 1),
+            Declaration(EVar(FreeVar(1)), GInt(5), freeCount = 1)
+          ),
+          body = GBool(false),
+          bindCount = 2
+        ),
+        Let(
+          declarations = List(
+            Declaration(EVar(FreeVar(0)), GInt(4), freeCount = 1),
+            Declaration(EVar(FreeVar(1)), GInt(5), freeCount = 1)
+          ),
+          body = GInt(4),
+          bindCount = 2
+        ),
+        Let(
+          declarations = List(
+            Declaration(
+              Connective(ConnAndBody(ConnectiveBody(List(EVar(FreeVar(0)), EVar(FreeVar(1)))))),
+              GInt(5),
+              freeCount = 2
+            )
+          ),
+          body = GString("Test"),
```

Comment:
If another review agrees that it's easier to understand the test with `GInt(5)` instead, then I'll change it 👍 Otherwise, I think that metric is too subjective to warrant modification.